cloudservice/aws/aws.go

```diff
@@ -49,25 +49,28 @@ func (s *CloudProviderIPRangeManager) ConvertToGenericIPRanges(unparsedIPRanges
 		return nil, err
 	}
 
-	genericIPRanges := types.CloudProviderIPRangeInfo{
-		CloudProviderServiceName: types.CloudProviderAWS,
-		Version:                  ipRanges.SyncToken,
-		IPRanges:                 make(map[string][]string),
+	result := &types.CloudProviderIPRangeInfo{
+		ServiceList: []string{},
+		IPRanges:    make(map[string][]string),
```

Comment:
we may need to strong-type the key of this map. Leaving it as string may cause surprise.

Reply:
We can't really have a static list of services in case gcp decide to add a new one. What would you suggest?

Reply:
on second read, isn't this map always just `Google Cloud`: [IPPrefixes]` ?

Reply:
Yes, whole explanation behind this in @luphaz comment!